src/combinatorics/simple_graph/basic.lean

```diff
@@ -537,66 +618,6 @@ end
 
 end finite
 
-section complement
-
-/-!
-## Complement of a simple graph
-
-This section contains definitions and lemmas concerning the complement of a simple graph.
--/
-
-/--
-We define `compl G` to be the `simple_graph V` such that no two adjacent vertices in `G`
-are adjacent in the complement, and every nonadjacent pair of vertices is adjacent
-(still ensuring that vertices are not adjacent to themselves.)
--/
-def compl (G : simple_graph V) : simple_graph V :=
-{ adj := λ v w, v ≠ w ∧ ¬G.adj v w,
-  sym := λ v w ⟨hne, _⟩, ⟨hne.symm, by rwa adj_comm⟩,
-  loopless := λ v ⟨hne, _⟩, false.elim (hne rfl) }
-
-instance has_compl : has_compl (simple_graph V) :=
-{ compl := compl }
-
-@[simp]
-lemma compl_adj (G : simple_graph V) (v w : V) : Gᶜ.adj v w ↔ v ≠ w ∧ ¬G.adj v w := iff.rfl
-
-instance compl_adj_decidable (V : Type u) [decidable_eq V] (G : simple_graph V)
-  [decidable_rel G.adj] : decidable_rel Gᶜ.adj := λ v w, and.decidable
-
-@[simp]
-lemma compl_compl (G : simple_graph V) : Gᶜᶜ = G :=
-begin
-  ext v w,
-  split; simp only [compl_adj, not_and, not_not],
-  { exact λ ⟨hne, h⟩, h hne },
-  { intro h,
-    simpa [G.ne_of_adj h], },
-end
-
-@[simp]
-lemma compl_involutive : function.involutive (@compl V) := compl_compl
```

Comment:
Did we lose this lemma and the one above? Or is it implied by things about general boolean algebras

Reply:
The `boolean_algebra` version of `compl_compl` is [here](https://leanprover-community.github.io/mathlib_docs/order/boolean_algebra.html#compl_compl). It doesn't look like we have `compl_involutive` for Boolean algebras, so maybe we should add it there? (either in this PR or another, I don't care.)

Reply:
`compl_compl` is a fact about boolean algebras: https://leanprover-community.github.io/mathlib_docs/order/boolean_algebra.html#compl_compl

But as far as I can tell, `compl_involutive` is missing, which should probably be added to `order.boolean_algebra`.

```lean
@[simp] theorem compl_involutive {α : Type*} [boolean_algebra α] :
  function.involutive (λ (x : α), xᶜ) := sorry
```

Reply:
The `sorry` is just `compl_compl`

Reply:
Yeah, this could be an issue, but it's not a `simple_graph` issue.

Reply:
#8357